### PR TITLE
Adding docstring in semcor

### PR DESCRIPTION
**Pull request by tejasvaidhyadev**
As per issue JuliaText#31. I am adding offline Docstring for SemCor and willing to do the same in others as well. Do you have any suggestions?

### CODE REVIEW

File: src/SemCor.jl [hunk 1]
```diff
@@ -1,7 +1,48 @@
 struct SemCor{S}
     filepaths::Vector{S}
 end
+"""
```
Comment (oxinabox) [inline src/SemCor.jl]:
```suggestion

"""
```

**Comment by oxinabox** [comment]
commits from #33 seem to be in here as well

**Comment by tejasvaidhyadev** [comment]
yeah
Should there be any changes?

**Comment by oxinabox** [comment]
for now just rebase off `master` and it should be sorted

**Comment by Ayushk4** [comment]
Something still seems to be wrong with git history. Can you look into it @tejasvaidhyadev ?

File: src/SemCor.jl [hunk 2]
```diff
@@ -2,6 +2,48 @@ struct SemCor{S}
...
+
+julia> index["abandon%2:38:00::"]
+2-element Array{Array{String,1},1}:
+ String["Jonathan", "wrote", "grimly", "of", "the", "destruction", "of", "Harpers_Ferry", "before", "they"  …  "First_Brigade", "had", "destroyed", "all", "the", "rolling_stock", "of", "the", "B_+_O_Railroad", "."]
+ String["The", "expense", "of_this", "type", "of", "organization", "in", "religious", "life", ","  …  "in", "congregations", "illumines", "the", "nature", "of", "the", "Protestant", "development", "."]
+
+
+```
+
+SemCor is a classical Sense Annotated Corpus. It Has a structure of documents, paragraphs, sentences, words. The words are either tagged with part of speech, or tagged with full lemma, part of speech and sensekey. We expose the word function to get back the words, and the sensekey function to extract the sense key.
```
Comment (Ayushk4) [inline src/SemCor.jl]: Reduce the length of the above line. - 

```
SemCor is a classical Sense Annotated Corpus. 
It has a structure of documents, paragraphs, sentences, words. 
The words are either tagged with part of speech, or tagged with full lemma, part of speech and sensekey. 
We expose the word function to get back the words, and the sensekey function to extract the sense key.
```

Also, maybe mention something about various SemCor versions.

Comment (tejasvaidhyadev) [reply src/SemCor.jl]: Thankyou @Ayushk4   for all your time and suggestion. I will Implement the above changes

**Comment by tejasvaidhyadev** [comment]
Can you review my PR  and should i add doc string for others @oxinabox and @Ayushk4

File: src/SemCor.jl [hunk 3]
```diff
@@ -1,7 +1,60 @@
...
+julia> index["abandon%2:38:00::"]
+2-element Array{Array{String,1},1}:
+ String["Jonathan", "wrote", "grimly", "of", "the", "destruction", "of", "Harpers_Ferry", "before", "they"  …  "First_Brigade", "had", "destroyed", "all", "the", "rolling_stock", "of", "the", "B_+_O_Railroad", "."]
+ String["The", "expense", "of_this", "type", "of", "organization", "in", "religious", "life", ","  …  "in", "congregations", "illumines", "the", "nature", "of", "the", "Protestant", "development", "."]
+
+```
+
+SemCor is a classical Sense Annotated Corpus.
+
+It Has a structure of documents, paragraphs, sentences, words.
```
Comment (oxinabox) [inline src/SemCor.jl]:
```suggestion
It has a structure of documents, paragraphs, sentences, words.
```

**Comment by oxinabox** [comment]
thanks

**Pull request merged**